Automattic/wp-calypso
# Jetpack products: Update product descriptions and features copy

## Changes proposed in this Pull Request

This PR updates descriptions and features copy of various Jetpack products.

These changes were requested here: p1HpG7-g2B-p2#comment-54581
Related to project: p1HpG7-g29-p2
Asana card: 1202316834912830-as-1202332690796700/f

## The updates are as follows:
(also described in [this doc](https://docs.google.com/document/d/1Qsp3zZO40fpnnAbmm0Zryd8oWU-alnyK69YhHg7ndDE)):

### Backup
**Update feature:**
 - Starts with 10GB of storage

### Site Search
**Add feature:**
- Works seamlessly with WooCommerce

### Jetpack CRM
**Update description:** Build better relationships with your customers and clients. Ideal for WooCommerce stores.
**Update feature:**
- Fully integrated with WordPress & WooCommerce

### Jetpack Free
**Add feature:** Downtime monitoring
**Add feature:** Related posts


### Boost Free
**Update description:** All of the essential tools to speed up your site — no developer required.
**Update features:**

- Optimize CSS loading
- Defer non-essential JavaScript
- Lazy image loading
- Site performance scores


### Social Free
**Update description:** Easily share your website content on your social media channels.
**Update features:**

- Auto-publish on popular social media platforms
- Schedule your posts
- Preview content before sharing
- Central dashboard to manage all social platform connections


## Testing instructions
 1. Checkout this PR and spin up Calypso green (`yarn start-jetpack-cloud`)
 2. Open http://jetpack.cloud.localhost:3000/pricing
 3. Check each product described above and check that the descriptions and features have been updated/added accordingly.
